lib/simulator-xcode-9.js

```diff
@@ -64,6 +67,13 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     if (opts.scaleFactor) {
       opts.devicePreferences.SimulatorWindowLastScale = parseFloat(opts.scaleFactor);
     }
+    if (opts.simulatorWindowCenter) {
+      if (validCoordinates(opts.simulatorWindowCenter)) {
+        opts.devicePreferences.SimulatorWindowCenter = opts.simulatorWindowCenter;
+      } else {
+        log.warn(`Invalid simulatorWindowCenter value: ${opts.simulatorWindowCenter}. Valid string format "{x,y}"`);
```

Comment:
... and the verification to be moved into _verifyDevicePreferences_ method

Reply:
@mykola-mokhnach where's the _verifyDevicePreferences_ method ?

Reply:
just rebase with the master branch